### ISSUE TITLE
Initialize PayPal and ApplePay with local braintree library

### ISSUE THREAD

**Issue by niklv** [recurly/recurly-js#873]
In my project we already have self-hosted braintree-web cleint from npm. It will be great to add option to initialize PayPal and ApplePay with my braintree library instanse, to skip 3rd party scripts loading in initialization.

**Comment by cbarton** [comment]
Recurly.JS will check if the module that is being requested (`client`, `applePay`, etc.) is available in the compatible version before loading the library via the 3rd party script.

Is the ask to provide the instance of the `braintree.client` that has already been initialized with the client authorization token?

**Comment by niklv** [comment]
No, I want to speed-up recurly initialization by providing already bundled braintree package. So there no need to load this lib from 3rd party. (I use official braintree-web npm package)

**Comment by cbarton** [comment]
Per my comment above, we will not load the 3rd party library should the [module be loaded already ](https://github.com/recurly/recurly-js/blob/master/lib/util/braintree-loader.js#L21-L25). 

Could you provide me with the BT library version you are using? I want to confirm that it does not match what[ RJS expects](https://github.com/recurly/recurly-js/blob/master/lib/util/braintree-loader.js#L21-L25) because that might be the reason why it is triggering the external load. If it doesn't match we might be able to loosen that version constraint since there isn't much in the way of breaking changes in the library in v3.
